settings.gradle

```diff
@@ -13,4 +13,8 @@ pluginManagement {
     }
 }
 
+plugins {
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.7.0'
+}
+
```

Comment:
Has to be defined in the `settings.gradle`

Makes sure the relevant JDK is automatically downloaded again. This was changed in a more recent version, it used to download the JDK automatically if it was missing--now we need this plugin